tests/subsys/fw_info/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ cmake_minimum_required(VERSION 3.8.2)
 
 # Cannot choose via CONFIG_SOC_NRF9160 because this has to be run before
 # boilerplate.cmake.
-get_filename_component(provision_path provision.hex ABSOLUTE)
-#get_filename_component(provision_path provision_nrf9160.hex ABSOLUTE)
+set(provision_hex provision.hex)
+#set(provision_hex provision_nrf9160.hex)
```

Comment:
Why is this comment included? Is it to make it easy to test both provision.hex?

Reply:
Yes